### PR TITLE
[DDCE-1792]:HOSS GOV.UK libraries Upgrade

### PR DESCRIPTION
**Pull request by anburoach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Web application providing internal HMRC staff with an interface to check custome
 
 ## Running the app locally
 
-    sm --start HOSS
+    sm --start HOSS_ALL -r
```
Comment (fraser-sprott) [inline README.md:15]: We no longer need to specify the -r anymore this is used by default if not supplied, don't need to change this though

File: app/uk/gov/hmrc/homeofficesettledstatus/controllers/AccessibilityStatementController.scala [hunk 2]
```diff
@@ -16,22 +16,20 @@
...
+import scala.concurrent.Future
+
 @Singleton
 class AccessibilityStatementController @Inject()(
   controllerComponents: MessagesControllerComponents,
-  layoutComponents: LayoutComponents)(implicit configuration: Configuration)
-    extends FrontendController(controllerComponents) {
-
-  val accessibilityStatementPage = new AccessibilityStatementPage(layoutComponents)
+  implicit val appConfig: AppConfig,
```
Comment (fraser-sprott) [inline app/uk/gov/hmrc/homeofficesettledstatus/controllers/AccessibilityStatementController.scala:30]: can you move the implicit val's to go last

File: app/uk/gov/hmrc/homeofficesettledstatus/views/AccessibilityStatementPage.scala.html [hunk 3]
```diff
@@ -14,66 +14,66 @@
...
-    <div class="grid-row">
-        <div class="column-two-thirds">
+    <h1 class="govuk-heading-xl">@messages("accessibility.title")</h1>
+    <p class="govuk-body">This accessibility statement explains how accessible this service is, what to do if you have difficulty using it, and how to report accessibility problems with the service.</p>
+    <p class="govuk-body">This page only contains information about the check settled status service, available at <a href="/check-settled-status">https://admin.tax.service.gov.uk/check-settled-status</a></p>
 
-            <h1 class="heading-xlarge">@messages("accessibility.title")</h1>
-            <p class="govuk-body-l">This accessibility statement explains how accessible this service is, what to do if you have difficulty using it, and how to report accessibility problems with the service.</p>
-            <p class="govuk-body">This page only contains information about the check settled status service, available at <a href="/check-settled-status">https://admin.tax.service.gov.uk/check-settled-status</a></p>
+    <h2 class="govuk-heading-l">Using this service</h2>
```
Comment (fraser-sprott) [inline app/uk/gov/hmrc/homeofficesettledstatus/views/AccessibilityStatementPage.scala.html:34]: should this not be govuk-heading-xl to keep the format the same as before

File: app/uk/gov/hmrc/homeofficesettledstatus/views/AccessibilityStatementPage.scala.html [hunk 4]
```diff
@@ -14,66 +14,66 @@
...
 
-            <ul class="govuk-list govuk-list--bullet">
-                <li>change colours, contrast levels and fonts</li>
-                <li>zoom in up to 300% without the text spilling off the screen</li>
-                <li>get from the start of the service to the end using just a keyboard</li>
-                <li>get from the start of the service to the end using speech recognition software</li>
-                <li>listen to the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
-            </ul>
+    <p class="govuk-body">We have also made the text in the service as simple as possible to understand.</p>
+    <p class="govuk-body"><a href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
```
Comment (fraser-sprott) [inline app/uk/gov/hmrc/homeofficesettledstatus/views/AccessibilityStatementPage.scala.html:47]: the link should have the govuk-link class

File: app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html [hunk 5]
```diff
@@ -14,126 +14,80 @@
...
-
-@scriptElement = {
-    <script src="@routes.Assets.at("javascripts/scripts.js")" type="text/javascript"></script>
-    <script src="@routes.Assets.at("javascripts/ga-events.js")" type="text/javascript"></script>
-}
+@(title: String, beforeContentHtml: Option[Html] = None, gaCode: Seq[String] = Seq.empty)(mainContent: Html)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @head = {
-    @if(configuration.get[String]("google-tag-manager.id") != "N/A") {
+    <script src='@routes.Assets.versioned("javascripts/ga-events.js")' type="text/javascript"></script>
```
Comment (fraser-sprott) [inline app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html:34]: This should be able to be replaced with the tracking snippet helper, though believe this is also enabled by default if the config is present

Comment (anburoach) [reply app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html:34]: this is an internal service  so we shouldn't implement tracking consent snippet

File: conf/messages [hunk 6]
```diff
@@ -12,6 +18,7 @@ generic.change=Change
 
 # Validation errors
 error.summary.heading=There is a problem
```
Comment (fraser-sprott) [inline conf/messages:20]: theres no need to duplicate this, standard pattern is that heading should be the same as the title unless if theres a name or something

File: app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html (hunk 5 above)
Comment (fraser-sprott) [reply app/uk/gov/hmrc/homeofficesettledstatus/views/govuk_wrapper.scala.html:34]: always forget about that one

**Pull request merged**